assets/src/edit-story/app/uploader/useUploader.js

```diff
@@ -72,7 +72,7 @@ function useUploader(refreshLibrary = true) {
 
     const promise = uploadMedia(file, additionalData);
     if (refreshLibrary) {
-      promise.finally(resetFilters);
+      promise.finally(resetWithFetch);
```

Comment:
Is there a value in waiting for upload to complete before resetting? Especially if we need to put a placeholder somewhere there?

Reply:
I think it will be solved in other PRs (instant preview, local mode).